its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -570,9 +585,7 @@ private static Integer getMeasureAsInteger(String componentKey, String metricKey
   }
 
   private static WsClient newWsClient() {
-    return WsClientFactories.getDefault().newClient(HttpConnector.newBuilder()
-      .url(ORCHESTRATOR.getServer().getUrl())
-      .build());
+    return TestUtils.newWsClient(ORCHESTRATOR);
```

Comment:
This method is duplicated in all of the tests, so I added it to the _TestUtils_ class. I haven't refactored the other tests yet; I'll do that later.